src/codec2.c

```diff
@@ -182,8 +179,9 @@ struct CODEC2 * codec2_create(int mode)
 	c2->Sn_[i] = 0;
     c2->fft_fwd_cfg = codec2_fft_alloc(FFT_ENC, 0, NULL, NULL);
     c2->fftr_fwd_cfg = codec2_fftr_alloc(FFT_ENC, 0, NULL, NULL);
-    make_analysis_window(&c2->c2const, c2->fft_fwd_cfg, c2->w,c2->W);
+
```

Comment:
Rather than changing every single reference to w[] and W[] in ```codec2/src/*.c```, how about just (optionally) set up c2->w and c2->W to be pointers to your ROM table versions somewhere in the codec 2 init.